.changeset/pink-dogs-scream.md

```diff
@@ -0,0 +1,5 @@
+---
+'@iota/apps-ui-kit': minor
+---
+
+navigate to search result on enter and on arrow up/down
```

Comment:
the implementation in the ui kit does not relate to navigation, I suggest updating this description to something more generic that explains what has really been added